cmd/state/main.go

```diff
@@ -111,8 +114,11 @@ func run(args []string, out output.Outputer) (int, error) {
 		return 1, fail
 	}
 
+	nonInteractive := strings.ToLower(os.Getenv(constants.NonInteractive)) == "true" ||
+		argsHaveNonInteractive(args) ||
+		!terminal.IsTerminal(int(os.Stdin.Fd()))
```

Comment:
Does this work for both Unix and Windows?

Reply:
Yes, it does.  Tested in powershell and cmd shell and bash.